hs_core/models.py

```diff
@@ -1560,6 +1560,16 @@ def rdf_triples(self, subject, graph):
     @classmethod
     def validate_coverage_type_value_attributes(cls, coverage_type, value_dict):
         """Validate values based on coverage type."""
+        def compute_longitude(key_name):
```

Comment:
This is just a suggestion, but I think it'd be simpler if we checked for values that needed to be computed, then ran the same validation for all the values.  So, basically, change the last elif to if?

Reply:
Made the change.